src/pages/EditorPage.js

```diff
@@ -0,0 +1,64 @@
+import Editor from "../components/Editor.js";
+import { getDocumentContent, updateDocument } from "../utils/api.js";
+import { getItem, removeItem, setItem } from "../utils/storage.js";
+import { debounce } from "../utils/debounce.js";
+import { ACTIVE } from "../constant/constant.js";
+
+export default class EditorPage {
+  constructor({ $target, onEdit }) {
+    this.$target = $target;
+    this.$page = this.$target.querySelector(".editor");
+    this.editorComponent = new Editor({
+      $target: this.$page,
+      initialState: { title: "", content: "" },
+      onSetState: (editorState) => {
+        if (location.pathname === "/") return;
+        const id = location.pathname.split("/")[2];
```

Comment:
```suggestion
        const [,,id] = location.pathname.split("/");
```
이런 형태로 사용할수도 있을것 같네요 :)